NuGet.Config

```diff
@@ -36,8 +36,6 @@
     </author>
     <author name="James Newton-King">
       <certificate fingerprint="A3AF7AF11EBB8EF729D2D91548509717E7E0FF55A129ABC3AEAA8A6940267641" hashAlgorithm="SHA256" allowUntrustedRoot="false" />
-        <!-- jamesnk = James Newton-King, author of Newtonsoft.Json-->
-        <owners>jamesnk</owners>
```

Comment:
The `author` section cannot contain `owners`. See: https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file for examples.